src/cms/templates/pages/page_tree.html

```diff
@@ -64,7 +64,8 @@ <h2 class="heading font-normal mb-2">{% trans 'Pages' %}</h2>
             <tr class="border-b border-solid border-gray-200">
                 <th class="text-sm text-left uppercase py-3 pl-4 pr-2 min"><input type="checkbox" id="bulk-select-all"></th>
                 <th class="text-sm text-left uppercase py-3 pl-2 pr-2 min">{% trans 'Hierarchy' %}</th>
-                <th class="text-sm text-left uppercase py-3 pl-2 pr-2">{% trans 'Status' %}</th>
+                <th class="text-sm text-left uppercase py-3 pl-2 pr-2 min">{% trans 'Status' %}</th>
+                <th class="text-sm text-left uppercase py-3 pl-2 pr-2">{% trans 'Tags' %}</th>
```

Comment:
```suggestion
                <th class="text-sm text-left uppercase py-3 pl-2 pr-2 min">{% trans 'Tags' %}</th>
```
I'd keep the `min` class in this column, because the labels are usually no taking away much space. You can prevent the labels from breaking into multiple lines by using the css class `whitespace-nowrap` for the tag cells.